### DISCUSSION TITLE
Can I change the name of i18n.js?

### DISCUSSION THREAD

**Discussion by ChristoRibeiro** [vinissimus/next-translate#549] [Q&A] [answered]
Great work on this project! 

I would like to keep a consistent on files names. Quick question:

Is it possible to change the name of i18n.js? Like next.i18n.js?

Or even better, put the i18n.js content inside the next.config.js file?

**Comment by aralroca** [comment, marked as answer]
no, it has to be i18n.js or i18n.json. For now there is this restriction.
